.github/workflows/pr.yml

```diff
@@ -114,7 +114,7 @@ jobs:
         run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
       # Fail when new errors appear
-      - name: Run ansible-test sanity/units
+      - name: Run ansible-test sanity/units/integration
```

Comment:
Is this change and the other one related to hack folder related to the new copy_and_render role?

Reply:
Yes, that's the first role to use an integration test so I had to modify the way to launch the tests.